sartography/spiff-arena
# Feature/guest form submission

This adds in support to complete specified human tasks with a guest user.

To test:
1. Create a process model with a manual or user task and click the allow guest checkbox
1. Start the process model with the same user and go to the process instance show page to get the guid of the human task
1. Use that guid to create a url like: "[domain]/tasks/[process_instance_id]/[task_guid]" and navigate there from a browser that is not logged into spiff
1. This user should be able to complete the task and will be given the next allowed guest task if there is one
1. If the guest user tries to navigate anywhere that is not the task show page, they should be taken to the login screen
